ci-operator/step-registry/cucushift/installer/check/ibmcloud/dedicatedhost/cucushift-installer-check-ibmcloud-dedicatedhost-commands.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    export region
+    echo "Try to login..."
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+ibmcloud_login
+
+#the file which saved the resource group of the pre created dedicated host group (just created when create the pre dedicated host, and not in Default group).
+dhgRGFile="${SHARED_DIR}/ibmcloud_resource_group_dhg"
+dh_file=${SHARED_DIR}/dedicated_host
+
+if [ -f ${dh_file} ]; then
+    dh=$(cat ${dh_file})
+    dhgRG=$(cat ${dhgRGFile})
+    echo "the pre created dedicated hosts ${dh} in ${dhgRG} resource group."
+
+    mapfile -t allIns < <(ibmcloud is dh ${dh} --output JSON | jq -r .instances[].name)
+
+    for vm in "${allIns[@]}"
+    do
+        if [[ ! $vm =~ -worker-[0-9]{1}-* && ! $vm =~ -master-[0-9]{1}$ ]]; then
+            echo "ERROR: unexpected vm ${vm} on " "${workerIns[@]}" ", worker vm is expected!!"
+            exit 1
+        fi
+    done
+else
+    infra_id=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+    dhgRG=${infra_id}
+    mapfile -t dhs < <(ibmcloud is dhs --resource-group-name ${dhgRG} -q | awk '(NR>1) {print $2}')
+    echo "Get the dedicated hosts which created by the cluster:" "${dhs[@]}" " in resource group : ${dhgRG}"
```

Comment:
```suggestion
    echo "Get the dedicated hosts which created by the installer:" "${dhs[@]}" " in resource group : ${dhgRG}"
```

Reply:
https://www.shellcheck.net/wiki/SC2145

Reply:
I think you can use `${dhs[*}` instead of `${dhs[@]}`